compiler/rustc_mir_transform/src/reveal_all.rs

```diff
@@ -46,6 +46,7 @@ impl<'tcx> MutVisitor<'tcx> for RevealAllVisitor<'tcx> {
                 .filter(|elem| !matches!(elem, ProjectionElem::OpaqueCast(_)))
                 .collect::<Vec<_>>(),
         );
+        self.super_place(place, _context, _location);
```

Comment:
We should probably also invoke `super_constant` in the `visit_constant` below.

technically the same with `visit_ty`, but `super_ty` does nothing 😆